policies/CLA.md

```diff
@@ -0,0 +1,65 @@
+# Individual Contributor License Agreement
+
+In order to clarify the intellectual property license granted with Contributions from any person or entity, Commonhaus Foundation
+("Project") must have a Contributor License Agreement ("CLA") on file that has been signed by each Contributor, indicating agreement
+to the license terms below. This license is for your protection as a Contributor as well as the protection of Project;
+it does not change your rights to use your own Contributions for any other purpose.
+
+You accept and agree to the following terms and conditions for Your present and future Contributions submitted to Project.
+Except for the license granted herein to Project and recipients of software distributed by Project, You reserve all right,
+title, and interest in and to Your Contributions.
+
+## Definitions
+
+1. "You" (or "Your") shall mean the copyright owner or legal entity authorized by the copyright owner that is making this
+Agreement with Project. For legal entities, the entity making a Contribution and all other entities that control, are
+controlled by, or are under common control with that entity are considered to be a single Contributor. For the purposes
+of this definition, "control" means (i) the power, direct or indirect, to cause the direction or management of such entity,
+whether by contract or otherwise, or (ii) ownership of fifty percent (50%) or more of the outstanding shares, or
+(iii) beneficial ownership of such entity.
+"Contribution" shall mean any original work of authorship, including any modifications or additions to an existing work,
```

Comment:
We could insert an empty line between these two lines.

Reply:
Make a PR? Happy to take that. I merged (ahead of lawyer review, etc.) so I could get the tooling working.

Reply:
There was an empty line and the markdown linter complained 🤷

Reply:
Should only complain about dangling white space at the end of the line.. or two blank lines in a row. Will check and fix.